pkg/products/monitoring/dashboards/criticalSLOAlerts.go

```diff
@@ -4181,8 +4212,9 @@ const MonitoringGrafanaDBCriticalSLOAlertsJSON = `{
 			}
 		]
 	},
+	"refresh": "10s",
 	"time": {
-		"from": "now-15m",
+		"from": "now-5m",
```

Comment:
From what I can tell, this is the default "poll rate" that the graphs will use to update themselves if no `instant` field is set, I might be wrong, just going off my observations.

Reply:
Looks like this sets the time range value? https://grafana.com/docs/grafana/latest/dashboards/time-range-controls/#common-time-range-controls

![from](https://user-images.githubusercontent.com/47980969/88552355-e7ceb700-d01b-11ea-82d0-3eee707612c9.png)

I'm not sure what is the ideal value is for this, if this is the case.

Reply:
So I think what this does is that it checks if the alert was firing in any period from now to 5 minutes ago then uses that to determine whether to update dashboards which don't use instant fields, not so much of a problem now though I think with the auto-refresh